### PR TITLE
Features/49

### PR DESCRIPTION
**Pull request by jmonterrubio**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Benefits

Production ready environment

### Possible Drawbacks

Beware the new config file format

### Applicable Issues

#49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,13 +80,13 @@ The url is formed by the host and port defined in the previous step.
...
 ```
-# The prom2teams endpoint to send HTTP POST requests to.
+// The prom2teams endpoint to send HTTP POST requests to.
 url: 0.0.0.0:8089/v2/<Connector1>
 ```
 
 ### Templating
 
-prom2teams provides a [default template](app/teams/template.j2) built with [Jinja2](http://jinja.pocoo.org/docs/2.9/) to render messages in Microsoft Teams. This template could be overrided using the 'templatepath' argument ('--templatepath <Jinja2 template file path>') during the application start.
+prom2teams provides a [default template](resources/templates/teams.j2) built with [Jinja2](http://jinja.pocoo.org/docs/2.9/) to render messages in Microsoft Teams. This template could be overrided using the 'templatepath' argument ('--templatepath <Jinja2 template file path>') during the application start.
```
Comment (dortegau) [inline README.md]: Jinja2 link should point to 2.10 version: http://jinja.pocoo.org/docs/2.10/ (as is defined in _requirements.txt_ file)

File: README.md [hunk 2]
```diff
@@ -93,16 +95,20 @@ Other optional fields are skipped and not included in the Teams message.
...
-<img src="assets/swagger.png" alt="Swagger UI" style="width: 600px;"/>
+<img src="assets/swaggerv2.png" alt="Swagger UI" style="width: 600px;"/>
 
 ## Testing
 
 To run the test suite you should type the following:
 
 ```bash
-# After cloning prom2teams :)
+// After cloning prom2teams :)
```
Comment (dortegau) [inline README.md]: AFAIK # is used to write inline comments in bash :) (https://linux.die.net/Bash-Beginners-Guide/sect_02_02.html)

File: config/logging.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  console:
+    class: logging.StreamHandler
+    level: DEBUG
+    formatter: simple
+    stream: ext://sys.stdout
+  file:
+    class : logging.handlers.RotatingFileHandler
+    formatter: simple
+    filename: /tmp/prom2teams.log
+    maxBytes: 2097152 #200MB
```
Comment (dortegau) [inline config/logging.yml]: 7 files * 200 MB per file: 1.4 GB 

Probably is too much

File: tests/test_server.py [hunk 4]
```diff
@@ -1,54 +0,0 @@
-import unittest
```
Comment (dortegau) [inline tests/test_server.py]: Why are tests removed from prom2teams? 😄

File: prom2teams/config.ini [hunk 5]
```diff
@@ -1,3 +0,0 @@
-[HTTP Server]
```
Comment (manuhortet) [inline prom2teams/config.ini]: Default .ini still missing. Just the ones in /tests were re-added.

File: tests/test_server.py (hunk 4 above)
Comment (jmonterrubio) [reply tests/test_server.py]: added again, but i don't like them at all :)

File: config/logging.yml (hunk 3 above)
Comment (jmonterrubio) [reply config/logging.yml]: updated

File: README.md (hunk 1 above)
Comment (jmonterrubio) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (jmonterrubio) [reply README.md]: updated

File: prom2teams/config.ini (hunk 5 above)
Comment (jmonterrubio) [reply prom2teams/config.ini]: this file is not mandatory, default properties are setted in config/settings.py :)

**Pull request merged**